data/sql/updates/pending_db_world/rev_1677089599224449400.sql

```diff
@@ -0,0 +1,4 @@
+--
+DELETE FROM `npc_trainer` WHERE `SpellID`=60969;
```

Comment:
Usually it's better to add 2 fields when deleting. Just to be a bit safer
```suggestion
DELETE FROM `npc_trainer` WHERE `SpellID`=60969 AND `ID` IN (201040, 201041);
```